### PR TITLE
DEPLOY-567: Update instructions for deploying the DBP on Windows

### PR DESCRIPTION
**Pull request by svidrascu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -330,4 +331,242 @@ You may find it helpful to see the Kubernetes resources visually which can be ac
...
+```
+
+Note: Make sure to leave a new line at the end before saving it. 
+
+### 11. Install alfresco-dbp
+
+Copy and paste the following block into your command line.
+  
+```bash
+helm install alfresco-incubator/alfresco-dbp `
```
Comment (rgauss) [inline README.md]: Shouldn't this be updated to pull in `minimal-values.yaml` now?

Comment (svidrascu) [reply README.md]: exactly :)
i Just figured it our after merging and reverted.

**Comment by svidrascu** [comment]
Sorry for doing this, but we are going to need this guide updated to the latest dns and value changes brought by BUILDER-96 in master.

I merged master here to get the latest updates. The changes are not pretty big but we do have some. 
1. Using values files instead of helm install --set --set
2. Using nip.io and wildcard dns for local setup

File: README.md [hunk 2]
```diff
@@ -359,4 +360,271 @@ You may find it helpful to see the Kubernetes resources visually which can be ac
...
+Do the following command to find your ipv4 address and copy it to your clipboard.
+
+```bash
+ipconfig
+```
+
+Note: If there are many ipv4 addresses, use the ipv4 address of the connection that you are currently using.
+
+
+Paste the ipv4 address at the end of the hosts file in C:\Windows\System32\drivers\etc, followed by "localhost-k8s" as in the following example.
```
[the remark this answers is not in the document]
Comment (aymanthefirst) [reply README.md]: Do you mean I should omit all of step 12 and 13?

File: minimal-values.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+global:
+  keycloak:
+    url: "http://alfresco-identity-service.10.244.50.45.nip.io/auth"
```
[the remark this answers is not in the document]
Comment (aymanthefirst) [reply minimal-values.yaml]: I pulled this down when following the instructions, but never deleted it again 😞  Thanks for noticing

File: README.md [hunk 4]
```diff
@@ -359,4 +360,271 @@ You may find it helpful to see the Kubernetes resources visually which can be ac
...
+kubectl create -f secret-name.yml --namespace $DESIREDNAMESPACE
+```
+
+### 9. Add remote chart repository to Helm configuration.
+
+```bash
+helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
+```
+
+### 10. Add Local DNS
```
[the remark this answers is not in the document]
Comment (aymanthefirst) [reply README.md]: It seems that in master this step is also still included as shown below. Should it be omitted? 

### 8. Add Local DNS

We will be forming a local dns with the use of nip.io. All you have to do it get your ip using the following command.

```bash
export LOCALIP=$(ipconfig getifaddr en0)
```

*Note:* Save this ip for later use.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (svidrascu) [reply README.md]: Yes, remove them.

**Pull request merged**